web/src/components/SaveMessage.test.js

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { create, act } from "react-test-renderer";
+import moment from "moment";
+import SaveMessage from "./SaveMessage";
+
+describe("<SaveMessage />", () => {
+  let subject;
+
+  describe('when saved less than 1 minute ago, it displays "Saved"', () => {
+    [
+      ["1 second ago", 1],
+      ["2 seconds ago", 2],
+      ["30 seconds ago", 30],
+      ["59 seconds", 59],
+    ].forEach(([testName, seconds]) => {
+      test(testName, () => {
+        const lastSaved = moment().subtract(seconds, "seconds");
+        // https://reactjs.org/docs/test-renderer.html#testrendereract
+        act(() => {
+          subject = create(<SaveMessage lastSaved={lastSaved} />);
+        })
+        expect(subject.toJSON()).toEqual("Saved");
```

Comment:
I'm going to open a tech debt issue to come back and think about these tests at some point. These particular ones are a decent candidate for snapshots, for example.

Certainly not something worth changing *now* and maybe not worth changing at all! But let's put a pin in it and figure it out later. 😄